### ISSUE TITLE
Notification for own comments

### ISSUE THREAD

**Issue by cboltz** [openSUSE/open-build-service#746]
Please add a way/option to receive a mail for _own_ comments on a SR.
(I'd like to have my own comments in my mail archive, not only comments from other people.)

For the user interface, this would be a checkbox "also notify me if the change was made by me".

Note: maybe this not only affects SR comments, but also other notification types.

**Comment by vpereira** [comment]
![Screenshot_2019-09-24_14-37-53](https://user-images.githubusercontent.com/37418/65512116-ea7eee00-ded8-11e9-8179-9b11501b5c45.png)

AFAICS it was already implemented, right? Can we close it @cboltz @hennevogel ?

**Comment by coolo** [comment]
I don't see that on your screenshot. Can you annotate?

**Comment by hennevogel** [comment]
Receive notifications of comments created on $something for which you are....

[ X ] Commenter

There is also [code](https://github.com/openSUSE/open-build-service/blob/master/src/api/app/models/event/comment_event.rb#L21). I guess this is done...

**Comment by coolo** [comment]
Yeah, but this is not about future comments but about your own comments.

And there is even [code](https://github.com/openSUSE/open-build-service/blob/master/src/api/app/models/event_subscription/find_for_event.rb#L22)
